frontend/wallet/src/render/components/TransactionCell.re

```diff
@@ -322,17 +334,66 @@ module InfoSection = {
   };
 };
 
+// Making a separate styles module here to collocate the styles in the below
+// make function closer to the layout
```

Comment:
If this file is too large to be able to navigate, should we be thinking about pulling one or two components out?

Reply:
Leaving this for now since we'll refactor this section too